www/include/monitoring/downtime/template/AddDowntime.ihtml

```diff
@@ -154,5 +154,26 @@
 
     initDatepicker("datepicker", "mm/dd/yy", "0");
 
+    $(document).ready(function() {
+        $(".datepicker").first().on('change', function (e) {
+            updateEndTime();
+        });
+        $(".timepicker").first().on('change', function (e) {
+            updateEndTime();
+        });
+        $("#duration").on('keypress', function (e) {
+            updateEndTime();
+        });
+        $("#duration_scale").on('change', function (e) {
+            updateEndTime();
+        });
+        $(".datepicker").last().on('change', function (e) {
```

Comment:
can you remove this part please.
I misunderstood what you meant when you told me "it needs to work both ways"
It's not both ways when changing start/end date. But both ways when changing start date/time